packages/core/src/components/blind/blind.scss

```
@@ -45,20 +45,18 @@
         color: var(--theme-color-std-text);
       }
-
       .blind-header-title-sublabel {
         color: var(--theme-color-soft-text);
       }
 
       .header-actions {
         --ix-icon-button-color: var(--theme-color-primary);
       }
-    } @else {
+    }@else {
```

Stylelint (17.13.0)

[error] Expected single space after "}" of `@if` statement (scss/at-if-closing-brace-space-after)

(scss/at-if-closing-brace-space-after)